int32 Brightness
int32 Contras
int32 Saturation
